webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric-association.js

```diff
@@ -186,9 +202,34 @@ class SakaiRubricAssociation extends RubricsElement {
     }
   }
 
+  handleAssociated() {
+	
+    if (this.isAssociated == 1) {
+
+      this.enabledClass = "enabled"
+      setTimeout(function () {
+        if (this.showSelfReportCheck === 'true') {
+		  
+          document.getElementById("new_assignment_peer_assessment_num_reviews").setAttribute('disabled', true);
+          document.getElementById("new_assignment_peer_assessment_num_reviews").value = 1;
+        }
+      }.bind(this));
+    } else {
+      this.enabledClass = "disabled"
+      setTimeout(function () {
+        if (this.showSelfReportCheck === 'true') {
+          document.getElementById("new_assignment_peer_assessment_num_reviews").removeAttribute('disabled');
+        }
+      }.bind(this));
+    }
+    this.dispatchEvent(new CustomEvent('state-details'));
+  }
+
   associate(e) {
     this.isAssociated = e.target.value == 1;
+    this.handleAssociated2();
```

Comment:
I'm not seeing handleAssociated2 in this PR.

Reply:
That was temporary code, sorry.